tests/phpunit/class-wc-stripe-account-test.php

```
@@ -597,4 +597,67 @@ public function test_reconfigure_webhooks_on_update_with_current_webhooks() {
+				'rk_live_old',
+				false,
+				[],
+			],
+			'missing/invalid webhook data is a no-op'                       => [
+				'',
+				'rk_live_new',
+				false,
+				[],
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_maybe_decommission_webhook is not named in camelCase. (undefined)

(CamelCaseMethodName)